components/application-mgt/org.wso2.carbon.device.application.mgt.common/src/main/java/org/wso2/carbon/device/application/mgt/common/PaginationRequest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@NishadiPrasangini licence year should be 2018. Please check for other files as well.